### PR TITLE
Fix broken links in README.md

### PR DESCRIPTION
**Pull request by Elliot67**
# Pull Request

## What does this PR do?
- fix some links in the readme

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

### CODE REVIEW

**Comment by curquiza** [comment]
@guimachiavelli can you review the PR?
Especially
- check that broken links are expected
- the removed section should indeed be removed

File: README.md [hunk 1]
```diff
@@ -101,7 +99,7 @@ Meilisearch is a search engine created by [Meili](https://www.welcometothejungle
 
 - For feature requests, please visit our [product repository](https://github.com/meilisearch/product/discussions)
 - Found a bug? Open an [issue](https://github.com/meilisearch/meilisearch/issues)!
-- Want to be part of our Discord community? [Join us!](https://discord.gg/meilisearch)
+- Want to be part of our Discord community? [Join us!](https://discord.meilisearch.com/?utm_campaign=oss&utm_source=github&utm_medium=meilisearch&utm_content=feedback)
```
Comment (guimachiavelli) [inline README.md:102]: @Strift, can you confirm these are the correct fathom UTMs for our discord links?

Comment (Strift) [reply README.md:102]: `utm_content` should be `contact`

**Review by Strift — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (guimachiavelli) [reply README.md:102]:
```suggestion
- Want to be part of our Discord community? [Join us!](https://discord.meilisearch.com/?utm_campaign=oss&utm_source=github&utm_medium=meilisearch&utm_content=contact)
```

**Review by guimachiavelli — APPROVED** [review]

**Review by curquiza — APPROVED** [review]
bors merge

**Pull request merged**